libmozdata/patchanalysis.py

```diff
@@ -547,21 +547,27 @@ def attachmenthandler(attachments, bugid):
         'uplift_accepted': False,
         'uplift_comment': None,
         'uplift_author': None,
+        'uplift_reviewer': None,
         'landing_delta': timedelta(),
         'response_delta': timedelta(),
         'release_delta': timedelta(),
     }
     approval_flag = 'approval-mozilla-' + channel
 
-    app_flags = filter(lambda f: f['name'] == approval_flag, [flag for a in bug['attachments'] for flag in a['flags']])
+    app_flags = list(filter(lambda f: f['name'] == approval_flag, [flag for a in bug['attachments'] for flag in a['flags']]))
```

Comment:
Maybe directly use a list comprehension here and get rid of filter

Reply:
👍